src/assets/footer-social-fb.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="10" height="20" viewBox="0 0 10 20"><defs><style>.a{fill:#202020;fill-rule:evenodd;}</style></defs><path class="a" d="M335.821,7259v-9h2.732l.446-4h-3.179v-1.948c0-1.03.026-2.052,1.466-2.052h1.458v-2.86a16.121,16.121,0,0,0-2.519-.14c-2.646,0-4.3,1.657-4.3,4.7v2.3H329v4h2.923v9Z" transform="translate(-329 -7239)"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="10" height="13" viewBox="0 0 10 20"><defs><style>.a{fill:#202020;fill-rule:evenodd;}</style></defs><path class="a" d="M335.821,7259v-9h2.732l.446-4h-3.179v-1.948c0-1.03.026-2.052,1.466-2.052h1.458v-2.86a16.121,16.121,0,0,0-2.519-.14c-2.646,0-4.3,1.657-4.3,4.7v2.3H329v4h2.923v9Z" transform="translate(-329 -7239)"/></svg>
```

Comment:
Don't change the `height` here since `height` works for the given `viewBox`... Remember if the `viewBox` is not `0 0 24 24` i.e. 24x24, You'll need to use [`SvgIcon`](https://v4.mui.com/components/icons/#svgicon) component as a wrappe and set the actual `viewBox` prop so that MUI knows how to resize it.

Reply:
@kilemensi 😞 Why do I keep forgetting this. Alright changing that in a bit